src/style.css

```diff
@@ -0,0 +1,134 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --background-color: #fff;
+  --accent-color: #00acdc;
+}
+
+html {
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card {
+  height: 406px;
+  width: 198px;
+  background-color: var(--background-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.img {
+  display: block;
+  height: 134px;
+  width: 160px;
+  margin: 32px auto 0;
+}
+
+.card__description {
+  margin-top: 40px;
+  margin-left: 16px;
+  margin-right: 16px;
+}
+
+.card__code {
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--secondary-color);
+  line-height: 14px;
+
+  margin-top: 4px;
+}
+
+.card__reviews {
+  margin-top: 16px;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--5 .stars__star,
+.stars--4 .stars__star:not(:last-child),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--1 .stars__star:first-child {
+  background-image: url(./images/star-active.svg);
```

Comment:
please check this part.  it seems you have a minor chance to make all stars 'active'. 
you should use something like: `.stars--2 .stars__star:nth-child(-n+2),`

also try https://css-tricks.com/examples/nth-child-tester/